aws-synthetics-canary/src/main/java/com/amazon/synthetics/canary/CreateHandler.java

```diff
@@ -12,87 +11,94 @@
 import software.amazon.awssdk.services.synthetics.model.CreateCanaryRequest;
 import software.amazon.awssdk.services.synthetics.model.ValidationException;
 import software.amazon.awssdk.services.synthetics.model.StartCanaryRequest;
-import software.amazon.awssdk.services.synthetics.model.InternalServerException;
-import software.amazon.awssdk.services.synthetics.model.GetCanaryRequest;
-import software.amazon.awssdk.services.synthetics.model.GetCanaryResponse;
 
 import software.amazon.cloudformation.exceptions.*;
 import software.amazon.cloudformation.proxy.*;
 
 public class CreateHandler extends BaseHandler<CallbackContext> {
     private static final int DEFAULT_CALLBACK_DELAY_SECONDS = 10;
-    private static final int CALLBACK_DELAY_SECONDS_FOR_RUNNING_STATE = 30;
     private static final int MAX_RETRY_TIMES = 30;
     private static final int DEFAULT_MEMORY_IN_MB = 960;
 
-    private Logger logger;
-    private AmazonWebServicesClientProxy clientProxy;
```

Comment:
It might make sense to keep the instance variables and refer to them instead of including them in every method call at least for variables used in multiple methods

Reply:
Well, I've gotten rid of most of the methods.